Wrapping/R/CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ configure_file(
 if(NOT SimpleITK_FORBID_DOWNLOADS)
   include(sitkExternalData)
 
-  file( GLOB_RECURSE content_links RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "*.md5" )
+  file( GLOB_RECURSE content_links RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "Packaging/SimpleITK/data/*.md5" )
```

Comment:
Please Specify the last argument as "${CMAKE_CURRENT_SOURCE_DIR}/Packaging/SimpleITK/data/*.md5"